### PR TITLE
Start for providing a general overview of what the bundle is for

### PR DESCRIPTION
**Pull request by kegl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
-# ramp-workflow
+# The RAMP ecosystem
+
+The [RAMP](http://www.ramp.studio) ecosystem contains two organizations and three libraries. The purpose of the bundle is to __define, build, manage, and optimize data analytics workflows__, typically on the top of open source machine learning libraries like [pandas](http://pandas.pydata.org), [scikit-learn](http://scikit-learn.org/), and [keras](https://github.com/fchollet/keras/tree/master/keras). The bundle consists of
+1. [ramp-workflow](https://github.com/paris-saclay-cds/ramp-workflow) (this library) containing resuable tools and scripts to define
+    1. [score types](rampwf/score_types) (metrics),
+    2. [workflows and workflow elements](rampwf/workflows) (trainable data analytics modules like a classifier or a feature extractor),
+    3. [cross-valudation schemes](rampwf/cv_schemes) (guiding the evaluation procedure of the workflow), and
+    4. data connectors (to feed the workflows from various data sources).
+2. [databoard](https://github.com/paris-saclay-cds/databoard), a library managing the frontend and the database of the [RAMP](http://www.ramp.studio) platform. (should may be renamed ramp-board)
```
Comment (agramfort) [inline README.md]: yes databoard is not a good name anymore.

ramp-board works for me

**Comment by agramfort** [comment]
maybe explain also in readme why there is a need to the different orgs?

File: README.md (hunk 1 above)
Comment (kegl) [inline README.md]: Done.

**Comment by agramfort** [comment]
great

you'll need to update the readme file for ramp-board
to make it clear how to use it and contribute to it

**Pull request merged**